Dúvida: Será que o coveralls está correto?

Pessoal, eu tenho uma dúvida!
Rodei os testes localmente e a cobertura do projeto está em 71%, com dois testes que não passam. 
Porém, na página principal do projeto, o coveralls mostra só 30%. 
Isso está correto?
Saída dos testes abaixo, mas dá pra ver no travis também.
Obrigado!

```
.F....F.
======================================================================
FAIL: test_build_checkout_params_with_all_params (tests.test_pagseguro.PagseguroTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/victor/opensource/original/python-pagseguro/tests/test_pagseguro.py", line 88, in test_build_checkout_params_with_all_params
    self.assertIn(key, self.pagseguro.data)
AssertionError: 'currency' not found in {'shippingAddressPostalCode': '06650030', 'itemId1': '0001', 'reference': 'REF', 'itemId2': '0002', 'senderName': u'Guybrush Treepwood', 'itemDescription2': 'Produto 2', 'itemDescription1': 'Produto 1', 'shippingAddressComplement': '5 andar', 'shippingAddressCity': 'Sao Paulo', 'shippingCost': '1234.56', 'shippingAddressState': 'SP', 'redirectURL': '/redirecionando/', 'senderAreaCode': 11, 'senderPhone': 5555555, 'shippingAddressDistrict': 'Jardim Paulistano', 'email': 'seu@email.com', 'itemWeight1': 200, 'itemWeight2': 200, 'senderCPF': '00000000000', 'itemAmount1': 354.2, 'senderEmail': 'guybrush@monkeyisland.com', 'extraAmount': 12.5, 'shippingAddressNumber': 1234, 'shippingType': 2, 'shippingAddressCountry': 'BRA', 'itemAmount2': 355.2, 'itemQuantity2': 1, 'token': '123456', 'itemQuantity1': 2, 'shippingAddressStreet': 'Av Brig Faria Lima', 'abandonURL': '/abandonando/', 'senderBornDate': '06/08/1650'}

======================================================================
FAIL: test_pagseguro_initial_attrs (tests.test_pagseguro.PagseguroTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/victor/opensource/original/python-pagseguro/tests/test_pagseguro.py", line 51, in test_pagseguro_initial_attrs
    self.assertIn('currency', self.pagseguro.data)
AssertionError: 'currency' not found in {'token': '123456', 'email': 'seu@email.com'}

Name                   Stmts   Miss  Cover   Missing
----------------------------------------------------
pagseguro                135     46    66%   14-16, 19-29, 34-41, 45-61, 78, 128, 156, 165, 167, 172, 176, 180-183, 187-188, 192-193
pagseguro.configs         19      1    95%   6
pagseguro.exceptions       2      0   100%   
pagseguro.utils           52     14    73%   10, 35, 40-48, 74-75, 77, 89
----------------------------------------------------
TOTAL                    208     61    71%   
----------------------------------------------------------------------
Ran 8 tests in 0.120s

```

--- a/.coveragerc
+++ b/.coveragerc
@@ -1,0 +1,7 @@
+[run]
+source = pagseguro
+
+[report]
+omit =
+    */python?.?/*
+    */site-packages/nose/*

